### PR TITLE
[ESIMD] rename slm_load4/slm_store4 to slm_load_rgba/slm_store_rgba

### PR DESCRIPTION
**Pull request by DenisBakhvalov**
Converted to draft. Will return back when the corresponding compiler patch will be merged.

### CODE REVIEW

**Comment by v-klochkov** [comment]
@kbobrovs - CI passed, Please review.

File: SYCL/ESIMD/api/slm_gather_scatter_rgba.cpp [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+int main(void) {
+  queue q(esimd_test::ESIMDSelector{}, esimd_test::createExceptionHandler());
+
+  auto dev = q.get_device();
+  std::cout << "Running on " << dev.get_info<info::device::name>() << "\n";
+
+  bool passed = true;
+  passed &= test<int, 8>(q);
```
Comment (kbobrovs) [inline SYCL/ESIMD/api/slm_gather_scatter_rgba.cpp:179]: I'd suggest to add char and uint16_t cases, these are usually the most problematic cases.

Comment (kbobrovs) [reply SYCL/ESIMD/api/slm_gather_scatter_rgba.cpp:179]: If they fail - quick solution would be JIRA + TODO in the test

Comment (v-klochkov) [reply SYCL/ESIMD/api/slm_gather_scatter_rgba.cpp:179]: I did not find these intrinsics in the ESIMD DOC: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/ExplicitSIMD/dpcpp-explicit-simd.md 

But the implementation exists only for 4-byte elements: https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp#L698
https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp#L730

Comment (kbobrovs) [reply SYCL/ESIMD/api/slm_gather_scatter_rgba.cpp:179]: OK, looks like we need JIRAs on this.
Approving this one.

**Review by kbobrovs — APPROVED** [review]

**Pull request merged**